app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -847,7 +866,7 @@ class SettingsFragment : Fragment(R.layout.fragment_settings_screen) {
             }
         }
         io {
-            var proxyName = name
+            var proxyName = Constants.SOCKS
```

Comment:
is it a mistake to set `proxyName` always to `Constants.SOCKS`?

Reply:
Renamed the method name to `insertSocks5ProxyEndpointDB`. That method is only responsible for adding entry for SOCKS5 proxy.